wscript

```diff
@@ -1,170 +1,65 @@
-import json
-import os
-import re
-
-from waflib.Configure import conf
+#
+# This file is the default set of rules to compile a Pebble application.
+#
+# Feel free to customize this to your needs.
+#
+import os.path
 
 top = '.'
 out = 'build'
 
 
 def options(ctx):
     ctx.load('pebble_sdk')
-
     ctx.load('aplite_legacy', tooldir='waftools')
     ctx.load('configure_appinfo', tooldir='waftools')
     ctx.load('pebble_sdk_version', tooldir='waftools')
 
-
 def configure(ctx):
-    ctx.load('pebble_sdk')
-
+    """
+    This method is used to configure your build. ctx.load(`pebble_sdk`) automatically configures
+    a build for each valid platform in `targetPlatforms`. Platform-specific configuration: add your
+    change after calling ctx.load('pebble_sdk') and make sure to set the correct environment first.
+    Universal configuration: add your change prior to calling ctx.load('pebble_sdk').
+    """
+    ctx.env.CFLAGS = ['-std=c11',
+                      '-fms-extensions',
+                      '-Wno-address',
+                      '-Wno-type-limits',
+                      '-Wno-missing-field-initializers']
+    ctx.load('pebble_sdk')  # Need to remove -std=c99
     ctx.configure_appinfo([ctx.appinfo_bitmap_to_png])
```

Comment:
AWESOME!!!!!! This also makes it much easier to convert pebblejs to a Pebble Package